src/components/card/Card/Card.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import classnames from 'classnames'
+
+interface CardProps {
+  header?: React.ReactNode
+  media?: React.ReactNode
+  footer?: React.ReactNode
+  standardLayout?: 'default' | 'headerFirst'
+  flagLayout?: 'default' | 'mediaOnRight'
+}
```

Comment:
I'm finding the naming of the layout props confusing, and also could be problematic if you can specify both a `standardLayout` and a `flagLayout`? What if the prop was like:
```
layout: 'standardDefault' | 'headerFirst' | 'flagDefault' | 'flagMediaRight'
```

Reply:
Thanks for finding this! I added a header first prop to the card component.